python-lib/font_exceptions_dict.py

```diff
@@ -2,26 +2,15 @@
 """Module with constants defining the wordcloud font exceptions for languages not supported by Noto Sans Display"""
 
 FONT_EXCEPTIONS_DICT = {
-    "ar": "NotoSansArabic-Regular.ttf",
-    "bn": "NotoSansBengali-Regular.ttf",
-    "fa": "NotoSansArabic-Regular.ttf",
-    "gu": "NotoSansGujarati-Regular.ttf",
-    "he": "NotoSansHebrew-Regular.ttf",
-    "hi": "NotoSansDevanagari-Regular.ttf",
-    "hy": "NotoSansArmenian-Regular.ttf",
-    "kn": "NotoSansKannada-Regular.ttf",
-    "ml": "NotoSansMalayalam-Regular.ttf",
-    "mr": "NotoSansDevanagari-Regular.ttf",
-    "ne": "NotoSansDevanagari-Regular.ttf",
-    "sa": "NotoSansDevanagari-Regular.ttf",
-    "si": "NotoSansSinhala-Regular.ttf",
-    "ta": "NotoSansTamil-Regular.ttf",
-    "te": "NotoSansTelugu-Regular.ttf",
-    "th": "NotoSansThai-Regular.ttf",
-    "ur": "NotoSansArabic-Regular.ttf",
+    "gu": "NotoSansMerged-Regular-2048upem.ttf",
+    "kn": "NotoSansMerged-Regular-2048upem.ttf",
+    "ml": "NotoSansMerged-Regular-2048upem.ttf",
+    "te": "NotoSansMerged-Regular-2048upem.ttf",
     "zh": "NotoSansCJKsc-Regular.otf",
     "language_column": "Arial Unicode.ttf",
 }
+
+
 """dict: Languages supported by spaCy: https://spacy.io/usage/models#languages
```

Comment:
Can you update the docstring of this constant to explain what it is and how it works (in particular the merged fonts)?

Reply:
Can you remove the previous docstring, which no longer applies?
```suggestion
```